src/store/modules/constants.js

```diff
@@ -4,11 +4,16 @@ export const backend_baseurl = process.env.VUE_APP_API_URL;
 if (!backend_baseurl) {
   throw 'Backend URL has not been specified';
 }
-export const backend_keyboards_url = `${backend_baseurl}/v1/keyboards`;
+export const keyboards_baseurl = process.env.VUE_APP_KEYBOARDS_URL;
+if (!keyboards_baseurl) {
```

Comment:
I would use `isUndefined(keyboards_baseurl)` and null would be a valid value. It can be imported via lodash if it's not already, but usually we don't do checks like that in this file since you should get cascading failures immediately.

Reply:
I was copying the pattern used on lines 3-6. Should I update that as well?